opencsp/common/lib/render/figure_management.py

```diff
@@ -358,6 +364,10 @@ def setup_figure_for_3d_data(
 ) -> RenderControlFigureRecord:
     """Create and setup a new RenderControlFigureRecord for rendering on a 3D graph.
 
+    Note that even through the returned figure_record will ensure that the associated plot is closed when the associated
+    view object is destructed, it is almost always better to close the figure as soon as it's not needed any more via
+    the figure_record.close() method.

```

Comment:
🎉
Noted. I should probably start heeding this note.

Reply:
Haha yes please. That will help make unit tests more reliable. :)